lib/NetworkClient.js

```diff
@@ -220,6 +220,7 @@ var NetworkClient = function(defaultConfigOverride){
         }
         xhr.open(method, url, true);
         _setRequestHeaders(xhr, method, url, data, config);
+        xhr.withCredentials = config.withCredentials;
```

Comment:
if I don't pass a boolean here and I try to pass something else like `null` , `undefined` ,`'a string'` or `0` , how will the browser react? will he always consider it as `false` ? can you please check